lib/Workflow/OperationType/Command.pm

```diff
@@ -134,10 +134,12 @@ sub initialize {
             my $prop = $class_meta->property_meta_for_name($param_name);
 
             if ($prop && $prop->{is_param}) {
-                if ($prop->default_value) {
+                if (defined $prop->default_value) {
                     $self->$param_name($prop->default_value);
+                } elsif ($prop->calculated_default) {
```

Comment:
I think the code was ok but it doesn't hurt to be explicit.  I think it was ok because it would either be `undef` or a code ref.